pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -777,7 +777,7 @@ func NewClusterConfig() *ClusterConfig {
 		KubernetesNetworkConfig: &KubernetesNetworkConfig{
 			IPFamily: DefaultIPFamily,
 		},
-		VPC: NewClusterVPC(),
+		VPC: NewClusterVPC(false),
```

Comment:
This... shouldn't be false here, should it?

Reply:
the default new cluster is ipv4, so the cluster should contain an ipv4 vpc